dataproc/create_cluster_json.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample walks a user through creating a Cloud Dataproc cluster using
+# the Python client library and a json configuration.
+
+# [START dataproc_create_cluster]
+
+import google.cloud.dataproc_v1beta2.proto.clusters_pb2 as clusters
+import google.protobuf.json_format as json_format
+from google.cloud import dataproc_v1beta2
+
+
+def create_cluster(cluster_json_path, region):
+    '''
```

Comment:
Re: docstring style and the contents

Can you do:

```python

def create_cluster(cliuster_json_path, region):
    """Creates a cluster with a given cluster config file.

    Args:
        cluster_json_path: a path to the cluster config file
        region: region where the cluster will be created
    """
```

Sorry to be picky, but we're using this style. See the [AUTHORING_GUIDE.md](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md) for more details.